client/components/forms/form-radios-bar/index.jsx

```diff
@@ -0,0 +1,45 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormRadioWithThumbnail from 'components/forms/form-radio-with-thumbnail';
+
+const FormRadiosBar = ( { isThumbnail, checked, onChange, items } ) => {
+	const addProps = ( Component, item, key ) =>
+		<Component key={ key } checked={ checked === item.value } onChange={ onChange } { ...item } />;
+
+	return (
+		<div className={ classnames( 'form-radios-bar', { 'is-thumbnail': isThumbnail } ) }>
+			{ items.map(
+				( item, i ) =>
+					isThumbnail
+						? addProps( FormRadioWithThumbnail, item, i )
+						: <FormLabel key={ i }>
```

Comment:
It'd be awesome if we could [avoid using the index as the key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) though I'm not sure if we have anything more suitable (`item.value` perhaps? or a combination of `item.value` and the index?).
I think that in the majority of cases the index would actually be fine to use since these elements would be rendered statically and without any extra radio buttons being added dynamically but if we can cover the edge cases now we may avoid some tough debugging down the line :)